### PR TITLE
vetextid logic

### PR DESCRIPTION
**Pull request by jperk51**
- convert utc to fileman format
- add facility id
- add expect to tests

**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary
Pass correct vetextId to FE for appts deeplinking

## Related issue(s)
department-of-veterans-affairs/va-mobile-app#7081 

## Testing done
Added new expect and changed cassette to test edge case

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by aherzberg — APPROVED** [review]
LGTM

**Review by aherzberg — APPROVED** [review]
LGTM

File: modules/mobile/spec/models/adapters/appointments_vaos_v2_adapter_spec.rb [hunk 1]
```diff
@@ -643,7 +643,7 @@ def appointment_data(index = nil)
                                                           'status' => 'CANCELLED',
                                                           'status_detail' => 'CANCELLED BY CLINIC',
                                                           'time_zone' => 'America/Denver',
-                                                          'vetext_id' => nil,
+                                                          'vetext_id' => '442;3170516.',
```
Comment (kpethtel) [inline modules/mobile/spec/models/adapters/appointments_vaos_v2_adapter_spec.rb:646]: Have we confirmed that this is what we want? Having a dot with nothing after it seems strange?

**Review by kpethtel — APPROVED** [review]

**Review by aherzberg — APPROVED** [review]
LGTM

**Review by kpethtel — APPROVED** [review]
LGTM

**Pull request merged**